main.go

```diff
@@ -27,8 +29,13 @@ func main() {
 	imports := make(map[string]bool, 0)
 	for _, file := range files {
 		n := file.Name()
-		if strings.Contains(n, ".go") && !strings.Contains(n, "_test") {
-			f, err := os.Open(fmt.Sprintf("%s/%s", *in, n))
+		if n == "vendor" {
```

Comment:
I would prefer skipping directories at all here instead of the vendor one.

Reply:
Still printing a message to the user if "vendor" was discovered?

Reply:
Nope :) User would see a compiling error later on for missing packages.